skbio/io/format/embl.py

```diff
@@ -1352,17 +1320,27 @@ def _serialize_sequence(obj, indent=5):
     obj : DNA, RNA, Sequence Obj
     '''
 
-    # magic numbers
-    n = 1
-    line_size = 60
+    # a flag to determine if I wrote header or not
+    flag_header = False
+
+    # magic numbers: there will be 60 letters (AA, bp) on each line
+    chunk_size = 60
+
+    # letters (AA, bp) will be grouped by 10: each group is divided by
+    # one space from each other
     frag_size = 10
 
-    # count bases in sequence
+    # this is the final line length
+    line_size = 80
```

Comment:
From uniprot manual 
```
... Columns 73-80 of each 
sequence line contain base numbers for easier reading and quick 
location of regions of interest. The numbers are right justified and indicate
the number of the last base on each line.
```
So if we consider `80-73+1=8` columns, the biggest sequence that could be included in embl file will be `pow(10,9)-1` bp. [Here](http://book.bionumbers.org/how-big-are-genomes/) is an article on biggest genomes: *Paris Japonica* and *P.aethiopicus* are not yet sequenced. Then the biggest is Wheat, which assembly is not yet complete and largest chromsome shold be ~700Mb. So, I don't think we could found an example of such type.